.circleci/config.yml

```diff
@@ -140,18 +144,19 @@ commands:
     steps:
       - save_cache:
           name: Save cargo package cache
-          key: cargo-package-cache-{{ checksum "Cargo.lock" }}
+          key: cargo-package-cash-{{ checksum "Cargo.lock" }}
```

Comment:
Are you sure?

Reply:
lol, definitely not. The PR is in draft, and I need to test it in CI in-place.  So I deliberately changed it to cause a cache miss.  Might as well name it $_$ :)